clients/python/llmengine/api_engine.py

```diff
@@ -60,13 +60,15 @@ def validate_api_key(cls):
             )
 
     @classmethod
-    def _get(cls, resource_name: str, timeout: int) -> Dict[str, Any]:
+    def _get(
+        cls, resource_name: str, timeout: int, headers: Optional[Dict[str, Any]] = None
```

Comment:
nit: should headers be Optional[Dict[str, str]]? https://requests.readthedocs.io/en/latest/user/quickstart/#custom-headers mentions values need to be string/bytestring/unicode